segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/StorageTestBase.java

```diff
@@ -40,13 +42,20 @@
     protected static final int APPENDS_PER_SEGMENT = 10;
     private static final int SEGMENT_COUNT = 4;
 
+    protected StorageMetricsBase metrics = null;
```

Comment:
This is not being initialized and will result in an NPE if a subclass doesn't set it. 
If we want to make this mandatory initialize it by calling an abstract method.